invenio_indexer/api.py

```diff
@@ -190,10 +190,13 @@ def process_bulk_queue(self):
                 routing_key=self.mq_routing_key,
             )
 
+            req_timeout = current_app.config['INDEXER_BULK_REQUEST_TIMEOUT']
+
             count = bulk(
                 self.client,
                 self._actionsiter(consumer.iterqueue()),
                 stats_only=True,
+                request_timeout=req_timeout
```

Comment:
please add comma `,` at the end of line so we get one line changes in the future